src/adapter/src/coord/message_handler.rs

```diff
@@ -346,4 +349,22 @@ impl<S: Append + 'static> Coordinator<S> {
         .await
         .expect("updating compute instance status cannot fail");
     }
+
+    #[tracing::instrument(level = "debug", skip_all)]
+    async fn message_linearize_reads(&mut self, pending_read_txns: Vec<PendingTxn>) {
+        // An empty catalog transaction will confirm our leadership before sending a response.
+        self.catalog_transact(None, Vec::new(), |_| Ok(()))
```

Comment:
@mjibson do you know if this will fail if we have the wrong epoch? I have not been in the loop for any of that work.

Reply:
I'm pretty sure the answer to this is no. When reading from the stash we'll use the in memory cache so we never actually go to the stash. Then when committing, since we have no changes we also don't send anything to the stash.

Reply:
I agree.